unit/src/main/cpp/robot.cpp

```diff
@@ -1,20 +1,26 @@
+#include <iostream>
 #include <robot/state.h>
 #include <robot/control.h>
 #include <subsystems/subsystemBase.h>
 #include <subsystems/vision.h>
 #include <subsystems/display.h>
 #include <subsystems/miscellaneous.h>
+#include "utils/daq.h"
 #include "spdlog/spdlog.h"
 #include "spdlog/sinks/stdout_color_sinks.h"
+#include <QCoreApplication>
 
-int main() {
+int main(int argc, char** argv) {
     auto logger = spdlog::stdout_color_mt("console");
     spdlog::set_default_logger(logger);
 
+    QCoreApplication app(argc, argv);
+
     spdlog::info("Starting robot");
     State *state = State::instance();
     Commands *commands = Commands::instance();
     Outputs *outputs = Outputs::instance();
+    DAQ *daq = DAQ::instance("cu.usbmodem13401");
     Configs::configure();
```

Comment:
never touch robot.cpp, not sure why this is here? Move to a separate subsystem. That port should also be in constants